### PR TITLE
Create article: azs-how-create-vmfromsnapshot-portal

### PR DESCRIPTION
**Pull request by wturner123**
Created new article as per JIRA ticket AS-1477.

### CODE REVIEW

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+title: How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+description: Provides help for creating a virtual machine from a managed disk snapshot using the portal on UKCloud for Microsoft Azure
+services: azure-stack
+author: William Turner
+reviewer: William Turner
+lastreviewed: 05/05/2020
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Virtual Machine from disk snapshot
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:11]: Title is: `How to create a virtual machine from a disk snapshot`
Maybe also have "a" before `disk snapshot`?

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+---
+title: How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+description: Provides help for creating a virtual machine from a managed disk snapshot using the portal on UKCloud for Microsoft Azure
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:3]: I'm not sure if we call it the `UKCloud Azure Stack Hub portal` anymore?

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+reviewer: William Turner
+lastreviewed: 05/05/2020
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Virtual Machine from disk snapshot
+toc_sub3:
+toc_sub4:
+toc_title: Create a virtual machine from snapshot - Portal
+toc_fullpath: Users/How To/Create a Virtual Machine from disk snapshot/azs-how-create-vmfromsnapshot-portal.md
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:15]: Title is: `How to create a virtual machine from a disk snapshot`
Maybe also have "a" before `disk snapshot`?

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+toc_mdlink: azs-how-create-vmfromsnapshot-portal.md
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:16]: maybe some more dashes? `azs-how-create-vm-from-snapshot-portal.md` ?

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+# How to create a virtual machine from a disk snapshot using the UKCloud Azure Stack Hub portal
+
+## Overview
+
+The UKCloud Azure Stack Hub portal allows you to create a snapshot of a managed disk. This snapshot can be used to create a new managed disk, which you can then create a virtual machine from. This article shows you how to take the snapshot, create a managed disk, and create a virtual machine.
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:23]: Are we are creating the virtual machine from the snapshot?

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - **Location** - This will be the location of the Azure Stack Hub.
+
+    - **Account type** - Choose either Standard (HDD) or Premium (SSD) storage.
+
+    - **Source type** - Select *Snapshot* from the dropdown.
+        - **Source snapshot** - Select the previously created snapshot.
+
+    - **Size** - This will be set to the size of the source disk that the snapshot was created from. You can increase this if necessary, but you will not be allowed to decrease it below the original size.
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:95]: Maybe the part about not being able to decrease it should be in a !NOTE or !WARNING. To draw more attention to it?

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+4. When you're done, click the **Create** button to create the disk.
+
+5. You can monitor the progress of the disk creation by clicking the **Notifications** icon.
+
+    ![Notification showing disk creation in progress](images/azs-browser-create-disk-from-snapshot-progress.png)
+
+## Creating a virtual machine from a managed disk
+
+After you have the managed disk that you want to use, you can create the VM in the portal:
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:107]: Hmm 🤔 Not sure about: `After you have the managed disk that you want to use`.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+1. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+2. From the list, select the disk that you would like to create a virtual machine from.
+
+    ![Disks blade](images/azs-browser-disks-list.png)
+
+3. In the new blade for the selected disk, under *Overview*, ensure that **Disk state** is listed as *Unattached*. If it isn't, you will need to either detach the disk from the VM or delete the VM to free up the disk.
```
Comment (dbrennand) [inline articles/azure/azs-how-create-vmfromsnapshot-portal.md:117]: Not sure if deleting the entire VM is necessary. Detaching is fine IMO.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 2 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: This seems to be common wording in other docs.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 5 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Reworded in latest commit.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 7 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Removed wording in latest commit.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 8 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Reworded in latest commit.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 6 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Updated in latest commit.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 4 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Renamed in latest commit.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 3 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Reworded in latest commit.

File: articles/azure/azs-how-create-vmfromsnapshot-portal.md (hunk 1 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vmfromsnapshot-portal.md]: Reworded in latest commit.

File: articles/azure/azs-how-create-vm-from-snapshot-portal.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+    ![Disks blade](images/azs-browser-disks-list.png)
+
+3. In the new blade for the selected disk, under *Overview*, ensure that **Disk state** is listed as *Unattached*. If it isn't, you will need to detach the disk from the VM before you can use it.
+
+    ![New disk blade - Unattached](images/azs-browser-disk-unattached.png)
+
+4. In the menu at the top of the blade, click **Create VM**.
+
+5. In the **Create virtual machine** blade, in the *Basics* step, enter a Virtual machine name and either select an existing Resource group or create a new one, then click **OK**.
```
Comment (niall-lefley) [inline articles/azure/azs-how-create-vm-from-snapshot-portal.md:124]:
```suggestion
5. In the **Create virtual machine** blade, in the *Basics* step, enter a Virtual machine name and either select an existing resource group or create a new one, then click **OK**.
```

Comment (dbrennand) [reply articles/azure/azs-how-create-vm-from-snapshot-portal.md]: Fixed.

File: articles/azure/azs-how-create-vm-from-snapshot-portal.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+    ![Create snapshot from disk - New Blade](images/azs-create-snapshot-disk.png)
+
+5. In the **Create snapshot** blade, enter the following information:
+
+    - **Name** - The name of the snapshot
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-vm-from-snapshot-portal.md:62]: Select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.

File: articles/azure/azs-how-create-vm-from-snapshot-portal.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+
+    ![Disks blade - Add](images/azs-browser-disks-list-add.png)
+
+3. In the **Create managed disk** blade, enter the following information:
+
+    - **Name** - The name of the disk
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-vm-from-snapshot-portal.md:92]: Select an existing resource group, or create a new one by clicking the **Create new** link and then typing a name for your new resource group in the pop out window.

File: articles/azure/azs-how-create-vm-from-snapshot-portal.md (hunk 10 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vm-from-snapshot-portal.md]: Thanks for highlighting this.

File: articles/azure/azs-how-create-vm-from-snapshot-portal.md (hunk 11 above)
Comment (wturner123) [reply articles/azure/azs-how-create-vm-from-snapshot-portal.md]: Thanks for highlighting this.

**Review by cblackuk — COMMENTED** [review]
@wturner123  can you please check if the images are still valid? If so, LGTM

**Review by cblackuk — CHANGES_REQUESTED** [review]
- Probably need to point to our Docs for Sizes. Which might need fixing as well (SS doc).
- Maybe we need to explain how to detach a disk.

**Comment by wturner123** [comment]
* Left the link to the MS doc about sizes, as it's better suited in this situation. Have updated the hyperlink text.
* Added a note about shutting down the VM to detach the disk.

**Review by cblackuk — APPROVED** [review]
LGTM

**Comment by wturner123** [comment]
@suehighmoor, A fair few changes made so probably worth looking over again before we merge it.

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**